package.json

```diff
@@ -7,6 +7,7 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "@walletconnect/web3-provider": "1.4.1",
```

Comment:
notice that there is no cap in the version ^ 
This means we are locking the version to this specific version